cli/src/machine/bucket.rs

```diff
@@ -240,7 +244,12 @@ pub async fn handle_bucket(
             )?;
             signer.set_sequence(sequence, &provider).await?;
 
-            let metadata: HashMap<String, String> = args.metadata.clone().into_iter().collect();
+            let mut metadata: HashMap<String, String> = args.metadata.clone().into_iter().collect();
+            if let Some(alias) = &args.alias {
```

Comment:
If the alias flag is present, insert it in the `metadata` hashmap. this means we overwrite if `alias` is already present